sdk/src/main/java/com/meniga/sdk/webservices/requests/StartRealmSync.java

```diff
@@ -0,0 +1,21 @@
+package com.meniga.sdk.webservices.requests;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+
+/**
+ * Copyright 2017 Meniga Iceland Inc.
+ * Created by agustk on 4.4.2018.
+ */
+public class StartRealmSync extends QueryRequestObject {
```

Comment:
You can use kotlin here to remove boilerplate code:
```
data class StartRealmSync(
  var realmUserId: Long = 0,
  var waitForCompleteMilliseconds: Long = 0
) : QueryRequestObject() {
  override fun getValueHash(): Long = hashCode().toLong()
}
```

Reply:
I'll try it! :)